index.html

```diff
@@ -769,6 +769,14 @@ <h2 class='informative'>Vary</h2>
           defined in [[RFC3230]] to <code>POST</code> and <code>PUT</code> requests for <a>LDP-NR</a>s.
         </p>
       </section>
+      <section id="persistence-fixity" class="informative">
+        <h2>Persistence Fixity</h2>
+        <p>
+          Persistence fixity is verified by application of the <code>Want-Digest</code> header defined in
```

Comment:
The language here sounds like this is the _only_ way to implement persistence fixity. And while this is certainly the way I would implement it, perhaps it would be better to use text more like:

> A pattern for verifying persistence fixity of resources may include the application of the <code>Want-Digest</code> ...